Wireframe/css/style.css

```diff
@@ -0,0 +1,111 @@
+body {
+  box-sizing: border-box;
+  padding: 10%;
+  height: 100%;
+  font-family: "Lato", sans-serif;
+  font-size: 1rem;
+  background-color: FFF5E0;
+  justify-content: left;
+}
+
+header {
+  justify-content: space-between;
+  text-align: center;
+  color: #141e46;
+}
+
+.git-logo {
+  width: 30%;
+  height: auto;
+}
+.git-header-title {
+  color: #141e46;
+  font-size: 4rem;
+  margin: 0;
+}
+
+section {
+  display: grid;
+  margin-top: 2rem;
+  text-align: center;
+  justify-content: center;
+}
+
+.first_child {
```

Comment:
I'm afraid this is not what the project wanted. The request was "Use the :first-child pseudo-class to style the first article so that it stands out from the others" Do you remember learning about CSS pseudo-classes? These aren't classes that you create, they're there and work a little diffrently from normal classes. Could you change this to use the :first-child pseudo-class please? If you need any help let me know